apps/web/app/[lang]/blog/components/PostListHeader.tsx

```diff
@@ -1,14 +1,37 @@
+'use client';
+
+import { BlogCategory } from '@api/blog/types/blog.types';
 import { PostSearch } from './PostSearch';
-import { PostTabs } from './PostTabs';
+import { Dispatch, SetStateAction } from 'react';
+import { Select } from 'ui';
```

Comment:
Please merge these two lines in only one
```suggestion
import { Select, Tabs } from 'ui/core';
```